2018-06-11T17:45:24.315Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'EMA' as the object already exists)
    at /home/raj_dhamsaniya/GitHub/AgriInput/agricultureinput-network/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-11T17:45:24.317Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'EMA' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'EMA' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/raj_dhamsaniya/GitHub/AgriInput/agricultureinput-network/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-11T17:45:24.746Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)
    at /home/raj_dhamsaniya/GitHub/AgriInput/agricultureinput-network/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-11T17:45:24.747Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '12345' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/raj_dhamsaniya/GitHub/AgriInput/agricultureinput-network/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
